### ISSUE TITLE
[a11y] Select aria-expanded is never false

### ISSUE THREAD

**Issue by NicholasNewlandsDR** [material-components/material-web#5360]
### What is affected?

Accessibility

### Description

This [line](https://github.com/material-components/material-web/blob/main/select/internal/select.ts#L403) should be 'true': 'false' instead of 'nothing'.

Currently this fails the basic [Chai A11y aXe](https://open-wc.org/docs/testing/chai-a11y-axe/) to.be.accessible() check.

### Reproduction

Add the element and try and test the accessibility using the method in the provided link.

### Workaround

When testing the accessibility of a form, you will need to test each element skipping over the select instead of simply running the test on a wrapping element.

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

Failing in 1.1.1

### Browser/OS/Node environment

Using the recommended [web-test-runner](https://modern-web.dev/docs/test-runner/overview/).

**Comment by e111077** [comment]
Word from our A11y people. omitting it is correct, but adding false is also correct in many cases

Currently refactoring select to add `"false"`. Will consider for other elements as requested as it passes Google's A11y review
